import-service/serverless.ts

```diff
@@ -0,0 +1,62 @@
+import type { AWS } from '@serverless/typescript';
+import {importProductsFile} from '@functions/importProductsFile';
+import * as process from 'process';
+import {importFileParser} from '@functions/importFileParser';
+
+const serverlessConfiguration: AWS = {
+  service: 'import-service',
+  frameworkVersion: '3',
+  plugins: [
+    'serverless-esbuild',
+    'serverless-dotenv-plugin',
+    'serverless-auto-swagger'
+  ],
+  provider: {
+    name: 'aws',
+    runtime: 'nodejs14.x',
+    region: 'eu-west-1',
+    apiGateway: {
+      minimumCompressionSize: 1024,
+      shouldStartNameWithService: true,
+    },
+    environment: {
+      AWS_NODEJS_CONNECTION_REUSE_ENABLED: '1',
+      NODE_OPTIONS: '--enable-source-maps --stack-trace-limit=1000',
+      REGION: process.env.REGION,
+      BUCKET: process.env.BUCKET
+    },
+    iamRoleStatements: [
+      {
+        Effect: 'Allow',
+        Action: ['s3:ListBucket'],
+        Resource: ['arn:aws:s3:::files-upload-bucket']
+      },
+      {
+        Effect: 'Allow',
+        Action: ['s3:*'],
+        Resource: ['arn:aws:s3:::files-upload-bucket/*']
+      }
+    ]
+  },
```

Comment:
Actually, I wanted to provide only **s3:ListBucket** access for **process.env.BUCKET** and other list of actions for structures like **/uploaded** and **/parsed**, so that's the reason for using two different rules.

Reply:
Oh yea, that is smart